src/main.cpp

```diff
@@ -155,7 +156,8 @@ unique_ptr<CommandTable> commands(shared_ptr<Root> root) {
                                      &ClientManager::fullscreen_complete}},
         {"pseudotile",     {clients, &ClientManager::pseudotile_cmd,
                                      &ClientManager::pseudotile_complete}},
-        {"tag_status",     print_tag_status_command},
+        {"tag_status",     {global_cmds, &GlobalCommands::tagStatusCommand,
+                                          &GlobalCommands::tagStatusCompletion}},
```

Comment:
My OCD is tingling with the off-by-one indent :D

Reply:
[x] fixed :)

Reply:
Thanks!